docs/usage/configuration-options.md

```diff
@@ -1893,17 +1893,18 @@ Just like the earlier `matchPackagePatterns` example, the above will configure `
 
 ### matchPaths
 
-Renovate will match `matchPaths` against both a partial string match or a minimatch glob pattern.
-If you want to avoid the partial string matching so that only glob matching is performed, wrap your string in `+(...)` like so:
+Renovate will match `matchPaths` against a minimatch glob pattern.
```

Comment:
> Renovate will match `matchPaths` against a minimatch glob pattern.

This sentence has the word `match` in it three times:

- `match`
- `matchPaths`
- `minimatch`

How about this?

```suggestion
Renovate finds the file(s) listed in `matchPaths` with a minimatch glob pattern.
```